TS7016: Could not find a declaration file for module

Linting is okay, but the TS build with `microbundle` fails 🤔 

I have a `/types` directory with two definition files and they are configured to be used by tsc with `typeRoots`.

https://github.com/AndyOGo/stylelint-declaration-strict-value/tree/master/types

https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/7f9475d9f701ac9a1657f33b1f59aa184ab9727a/tsconfig.build.json#L8

```sh
> microbundle --tsconfig tsconfig.build.json --compress

(rpt2 plugin) Error: /home/travis/build/AndyOGo/stylelint-declaration-strict-value/src/index.ts(3,22): semantic error TS7016: Could not find a declaration file for module 'shortcss'. '/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/shortcss/lib/index.js' implicitly has an 'any' type.

  Try `npm install @types/shortcss` if it exists or add a new declaration (.d.ts) file containing `declare module 'shortcss';`

Error: /home/travis/build/AndyOGo/stylelint-declaration-strict-value/src/index.ts(3,22): semantic error TS7016: Could not find a declaration file for module 'shortcss'. '/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/shortcss/lib/index.js' implicitly has an 'any' type.

  Try `npm install @types/shortcss` if it exists or add a new declaration (.d.ts) file containing `declare module 'shortcss';`

    at error (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup/dist/shared/node-entry.js:5400:30)

    at throwPluginError (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup/dist/shared/node-entry.js:11878:12)

    at Object.error (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup/dist/shared/node-entry.js:12912:24)

    at Object.error (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup/dist/shared/node-entry.js:12081:38)

    at RollupContext.error (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:17194:30)

    at /home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24958:23

    at arrayEach (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:534:11)

    at forEach (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:9360:14)

    at printDiagnostics (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:24931:5)

    at Object.transform (/home/travis/build/AndyOGo/stylelint-declaration-strict-value/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:27374:17)
```

https://travis-ci.com/github/AndyOGo/stylelint-declaration-strict-value/builds/218179712#L2578-L2593

--- a/README.md
+++ b/README.md
@@ -223,6 +223,8 @@ Just point the input to a `.ts` file through either the cli or the `source` key 
 
 Microbundle will generally respect your TypeScript config defined in a `tsconfig.json` file with notable exceptions being the "[target](https://www.typescriptlang.org/tsconfig#target)" and "[module](https://www.typescriptlang.org/tsconfig#module)" settings. To ensure your TypeScript configuration matches the configuration that Microbundle uses internally it's strongly recommended that you set `"module": "ESNext"` and `"target": "ESNext"` in your `tsconfig.json`.
 
+To ensure Microbundle does not process extraneous files, by default it only includes your entry point. If you want to include other files for compilation, such as ambient declarations, make sure to add either "[files](https://www.typescriptlang.org/tsconfig#files)" or "[include](https://www.typescriptlang.org/tsconfig#include)" into your `tsconfig.json`.
+
 If you're using TypeScript with CSS Modules, you will want to set `"include": ["node_modules/microbundle/index.d.ts"]` in your `tsconfig.json` to tell TypeScript how to handle your CSS Module imports.
 
 ### CSS and CSS Modules

